File: BackEnd/src/main/java/com/salam/backend/enumeration/EtatConversationChatBot.java
package com.salam.backend.enumeration;

public enum EtatConversationChatBot {
    Ouverte, //La conversation est en cours.
    Fermée,  //Le problème a été résolu et la conversation est terminée.
    Escaladée, // La demande a été transférée à un expert humain ou à un niveau supérieur.
}

File: FrontEnd/src/app/enum/EtatConversationChatBot.ts
export enum EtatConversationChatBot {
  OUVERTE = 'Ouverte',      // La conversation est en cours.
  FERME = 'Fermée',        // Le problème a été résolu et la conversation est terminée.
  ESCALADEE = 'Escaladée'   // La demande a été transférée à un expert humain ou à un niveau supérieur.
}
